packages/api-explorer/__tests__/Doc.test.jsx

```diff
@@ -39,39 +40,27 @@ function assertDocElements(component, doc) {
   expect(component.find(`#page-${doc.slug}`).length).toBe(1);
 }
 
-test('should output correct components', () => {
+test('should output correct components', (done) => {
   const wrapper = mountWithIntl(
     <IntlProvider>
       <Doc {...props} />
     </IntlProvider>
   );
-
-  assertDocElements(wrapper, props.doc);
-  expect(wrapper.find(`div#page-${props.doc.slug}`).length).toBe(1);
-  expect(wrapper.find('PathUrl').length).toBe(1);
-  expect(wrapper.find('CodeSample').length).toBe(1);
-  // This test needs the component to be `mount()`ed
-  // but for some reason when I mount in this test
-  // it makes the test below that uses `jest.useFakeTimers()`
-  // fail ¯\_(ツ)_/¯. Skipping for now
-  expect(wrapper.find('Params').length).toBe(1);
-  expect(wrapper.find(ContentWithTitle)).toHaveLength(5);
-  expect(wrapper.find(SchemaTabs)).toHaveLength(1)
-});
-
-test('should render straight away if `appearance.splitReferenceDocs` is true', () => {
-  const doc = mountWithIntl(
-    <IntlProvider>
-      <Doc
-        {...props}
-        appearance={{
-          splitReferenceDocs: true,
-        }}
-      />
-    </IntlProvider>
-  );
-
-  expect(doc.find('Waypoint').length).toBe(0);
+  setTimeout(() => {
+    wrapper.mount()
+    assertDocElements(wrapper, props.doc);
+    expect(wrapper.find(`div#page-${props.doc.slug}`).length).toBe(1);
+    expect(wrapper.find('PathUrl').length).toBe(1);
+    expect(wrapper.find('CodeSample').length).toBe(1);
+    // This test needs the component to be `mount()`ed
```

Comment:
This is a nice comment but is it still relevant?

Reply:
removed